### PR TITLE
Move to new PacketIOActivity fields based on Instant

### PR DESCRIPTION
**Pull request by bbaldino**
Fun fact: `Instant.MIN`, which we use for our definition of `NEVER` is unable to be converted to a number of milliseconds-since-epoch.  This is probably good for `NEVER`, but broke some things here, so had to migrate away from the usage of the old properties and use `Instant` everywhere.

I think `EndpointConnectionStatus` could really use a unit test for its logic, but it'll need to be refactored so we can swap in a fake executor service, etc. so left that for future work.

### CODE REVIEW

**Comment by bbaldino** [comment]
> LGTM.
> 
> Does this mean we can remove the deprecated flavors of the APIs from jmt?

Yeah, believe so.  Do you think I should do that change and update the JMT version in the pom as part of this PR?

**Comment by JonathanLennox** [comment]
> > Does this mean we can remove the deprecated flavors of the APIs from jmt?
> 
> Yeah, believe so. Do you think I should do that change and update the JMT version in the pom as part of this PR?

Probably yes, to make sure we didn't miss something using the deprecated APIs.

**Comment by bbaldino** [comment]
> > > Does this mean we can remove the deprecated flavors of the APIs from jmt?
> > 
> > 
> > Yeah, believe so. Do you think I should do that change and update the JMT version in the pom as part of this PR?
> 
> Probably yes, to make sure we didn't miss something using the deprecated APIs.

And indeed there was!  Will open the JMT PR, too.

File: src/main/java/org/jitsi/videobridge/Endpoint.java [hunk 1]
```diff
@@ -214,34 +219,37 @@
...
-                logger,
-                Clock.systemUTC());
+        transceiver = new Transceiver(
+            id,
+            TaskPools.CPU_POOL,
+            TaskPools.CPU_POOL,
+            TaskPools.SCHEDULED_POOL,
+            diagnosticContext,
+            logger,
+            Clock.systemUTC()
```
Comment (JonathanLennox) [inline src/main/java/org/jitsi/videobridge/Endpoint.java]: Presumably you should now pass `clock` to `Transceiver` rather than using `Clock.systemUTC()` explicitly?

File: src/main/java/org/jitsi/videobridge/Endpoint.java [hunk 2]
```diff
@@ -104,8 +104,7 @@
     /**
      * The time at which this endpoint was created (in millis since epoch)
      */
-    //TODO: use clock/instant in this file
-    private final Long creationTimeMillis;
+    private final Instant creationTimeMillis;
```
Comment (JonathanLennox) [inline src/main/java/org/jitsi/videobridge/Endpoint.java]: Shouldn't have `Millis` in the name if it's an `Instant`.

File: src/main/java/org/jitsi/videobridge/Endpoint.java [hunk 3]
```diff
@@ -622,21 +621,17 @@ public boolean shouldExpire()
...
+
+        Instant lastActivity = getLastActivity();
+        Instant now = clock.instant();
+        if (lastActivity == ClockUtils.NEVER)
         {
-            Duration timeSinceCreation = Duration.ofMillis(now - creationTimeMillis);
+            Duration timeSinceCreation = Duration.between(creationTimeMillis, now);
             if (timeSinceCreation.compareTo(EP_TIMEOUT) > 0) {
                 logger.info("Endpoint's ICE connection has neither failed nor connected " +
                     "after " + timeSinceCreation + ", expiring");
```
Comment (JonathanLennox) [inline src/main/java/org/jitsi/videobridge/Endpoint.java]: Now we're getting to the point where we might want a more human-friendly equivalent of `Duration.toString()`, but that can be for a separate patch.

Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/Endpoint.java]: sounds good
